common-ktx/src/main/java/co/nimblehq/common/extensions/ToastExt.kt

```diff
@@ -0,0 +1,68 @@
+package co.nimblehq.common.extensions
+
+import android.app.Activity
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.widget.TextView
+import android.widget.Toast
+import androidx.annotation.IdRes
+import androidx.annotation.LayoutRes
+import androidx.fragment.app.Fragment
+
+/**
+ * Provide extension functions to Activity/Fragment to capable of showing Toast in a shorter call
+ *
+ * @param message A message to display
+ * @param duration Duration time to display message
+ */
+fun Activity.toastShort(message: CharSequence, duration: Int = Toast.LENGTH_SHORT) {
+    Toast.makeText(this, message, duration).show()
+}
+
+fun Fragment.toastShort(message: CharSequence, duration: Int = Toast.LENGTH_SHORT) {
+    Toast.makeText(context, message, duration).show()
+}
+
+fun Activity.toastLong(message: CharSequence, duration: Int = Toast.LENGTH_LONG) {
+    Toast.makeText(this, message, duration).show()
+}
+
+fun Fragment.toastLong(message: CharSequence, duration: Int = Toast.LENGTH_LONG) {
+    Toast.makeText(context, message, duration).show()
+}
+
+/**
+ * Provide extension functions to Activity to capable of showing Toast with layout
+ *
+ * Sample:
+ *     toastWithLayout(
+ *         message = "message",
+ *         layoutRes = R.layout.view_pin_toast_error,
+ *         rootLayoutRes = R.id.toastRootLayout,
+ *         textViewId = R.id.tvErrorMessage
+ *     )
+ *
+ * @param message A message to display
+ * @param layoutRes Layout resource id of the layout
+ * @param rootLayoutRes Id of the root ViewGroup
+ * @param textViewId Id of TextView that will be set message
+ */
+fun Activity.toastWithLayout(
```

Comment:
We can't apply this extension for Fragment? 🤔

Reply:
I'm afraid not. 🤔

Reply:
@lydiasama @hoangnguyen92dn We can do the `Toast` even with `context` afaik 😄

Reply:
@doannimble I think we could get rid of custom view for Toast. It's deprecated on `setView` method.

> Custom toast views are deprecated. Apps can create a standard text toast with the makeText(Context, CharSequence, int) method, or use a Snackbar when in the foreground. Starting from Android Build.VERSION_CODES.R, apps targeting API level Build.VERSION_CODES.R or higher that are in the background will not have custom toast views displayed.